x-pack/plugins/endpoint/public/applications/endpoint/types.ts

```diff
@@ -75,17 +75,67 @@ export interface PolicyListState {
 }
 
 /**
- * Policy list store state
+ * Policy details store state
  */
 export interface PolicyDetailsState {
   /** A single policy item  */
-  policyItem: PolicyData | undefined;
+  policyItem?: PolicyData;
   /** data is being retrieved from server */
+  policyConfig?: PolicyDetailsConfig;
   isLoading: boolean;
   /** current location of the application */
   location?: Immutable<EndpointAppLocation>;
 }
 
+export interface PolicyDetailsConfig {
```

Comment:
We probably want to call this type `PolicyConfig` since its not going to be specific to the Policy details, but more across our entire app.